packages/manager/modules/pci/src/projects/project/ai/apps/dashboard/dashboard.html

```diff
@@ -1,42 +1,62 @@
-<oui-header
-    data-heading="{{ $ctrl.app.spec.name }}"
-    description="{{ $ctrl.app.id }}"
->
-    <oui-header-tabs>
-        <oui-header-tabs-item
-            data-href="{{:: $ctrl.generalInformationLink }}"
-            data-active="$ctrl.currentActiveLink() === $ctrl.generalInformationLink"
-            ><span data-translate="pci_apps_dashboard_info_tab_title"></span>
-        </oui-header-tabs-item>
-        <oui-header-tabs-item
-            data-href="{{:: $ctrl.attachDataLink }}"
-            data-active="$ctrl.currentActiveLink() === $ctrl.attachDataLink"
-            ><span
-                data-translate="pci_apps_dashboard_attach_data_tab_title"
-            ></span>
-        </oui-header-tabs-item>
-    </oui-header-tabs>
-    <oui-guide-menu data-text="{{:: 'pci_project_guides_header' | translate }}">
-        <oui-guide-menu-item
-            data-ng-repeat="guide in $ctrl.guideUrl.ai_machine_learning track by $index"
-            data-href="{{:: guide.url }}"
-            data-on-click="$ctrl.trackClick($ctrl.guideTrackingSectionTags.ai_apps + guide.tracking)"
-            data-external
+<header class="oui-header">
+    <div class="oui-header__container">
+        <div class="oui-header__content">
```

Comment:
It's you don't have particular custom need, it's more efficient use the webComponent <oui-header></oui-header>